ci/travis-after-success.sh

```diff
@@ -1,6 +1,9 @@
 #!/bin/sh
 set -xe
 
+# Submit code coverage report
+curl -s https://codecov.io/bash | bash
```

Comment:
Will try `bash -x`. Note though that credential issue is a major vulnerability in all public CI services - anyone can make PR against your repo with arbitrary modifications to `.travis.yml` and unless you manage to stop the build before it gets tested, the script will run with your credentials. I am not aware of any solutions to that (though of course it is not as subtle as changing remote bash script).